Autoscaling APIs not experimental (#67202)
elastic/elasticsearch: docs/reference/autoscaling/deciders/machine-learning-decider.asciidoc

[role="xpack"]
[testenv="enterprise"]
[[autoscaling-machine-learning-decider]]
=== Machine learning decider

The machine learning decider (`ml`) calculates the required memory to run the
set of machine learning jobs created by users.

It is enabled for policies governing `ml` nodes and has following
configuration options:

* `num_anomaly_jobs_in_queue`, default 0
* `num_analytics_jobs_in_queue`, default 0

> * `down_scale_delay`, default 1 hour

Author: I will re-arrange and then ask ML team to add more detailed definitions in a follow-up.

> * `down_scale_delay`, default 1 hour

Author: See d0429bd4c243159e1571238588e123dde1eeedf4

[role="xpack"]
[testenv="enterprise"]
[[autoscaling-machine-learning-decider]]
=== Machine learning decider

The {ml} decider (`ml`) calculates the memory required to run
{ml} jobs created by users.

The {ml} decider is enabled for policies governing `ml` nodes.

[[autoscaling-machine-learning-decider-settings]]
==== Configuration settings

`num_anomaly_jobs_in_queue`::
(Optional, integer)

> Number of queued anomaly jobs to allow. Default 0.

Reviewer: ```suggestion
Number of queued anomaly jobs to allow. Defaults to `0`.
```

> `num_analytics_jobs_in_queue`::
> Number of queued analytics jobs to allow. Default 0.

Reviewer: ```suggestion
`num_analytics_jobs_in_queue`::
(Optional, integer)
Number of queued analytics jobs to allow. Defaults to `0`.

```

`down_scale_delay`::

> (Optional, <<time-units,time value>>

Reviewer: ```suggestion
(Optional, <<time-units,time value>>)
```

> Delay before scaling down. Default 1 hour.

Reviewer: ```suggestion
Delay before scaling down. Defaults to 1 hour.
```
